elyra/cli/pipeline_app.py

```diff
@@ -549,11 +565,12 @@ def describe(json_option, pipeline_path):
             else:
                 click.echo(f"{readable_key}: {describe_dict.get(key, blank_field)}")
     else:
-        for key in iter_keys:
-            describe_dict[key] = list(describe_dict[key])
-        for key in pipeline_keys:
+        for key in list(describe_dict.keys()):
+            if type(describe_dict[key]) is set:
+                describe_dict[key] = list(describe_dict[key])
+
             value = describe_dict.get(key)
-            if value is None or (key in iter_keys and len(value) == 0):
+            if value is None or (key in list(describe_dict.keys()) and len(str(value)) == 0):
```

Comment:
Not: this should yield the same result for `None` and '':

```suggestion
            if not value:
```

Reply:
Done